### ISSUE TITLE
0.29.0 release not pushed to pypi

### ISSUE THREAD

**Issue by achantavy** [lyft/cartography#460]
*Description*:
> What issue is being seen?  Describe what should be happening instead of the bug, for example: Cartography should not crash, the expected value isn't returned, the data schema is wrong, etc.

The cartography 0.29.0 release has failed to push to pypi.

*To Reproduce*:
> Steps to reproduce the behavior.  Provide all data and inputs required to reproduce the issue.

Draft a release.

*Logs*:
> If applicable, copy and paste your console log with the failing stack trace.

CI link : https://travis-ci.org/github/lyft/cartography/builds/743003897
then expand the last error msg:

```
Writing cartography-0.29.0/setup.cfg
creating dist
Creating tar archive
removing 'cartography-0.29.0' (and everything under it)
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.6.7/bin/twine", line 8, in <module>
    sys.exit(main())
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/twine/__main__.py", line 28, in main
    result = cli.dispatch(sys.argv[1:])
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/twine/cli.py", line 80, in dispatch
    main = registered_commands[args.command].load()
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2446, in load
    self.require(*args, **kwargs)
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2469, in require
    items = working_set.resolve(reqs, env, installer, extras=self.extras)
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/pkg_resources/__init__.py", line 775, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (importlib-metadata 0.18 (/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages), Requirement.parse('importlib-metadata>=1; python_version < "3.8"'), {'keyring'})
```

*Please complete the following information:*:
>  - Cartography release version or commit hash [e.g. 0.12.0 or 95e8e11913e2a44a4d4682506d8364a638ceac69]

0.29.0

*Additional context*:
> Add any other context about the problem here.
- https://travis-ci.community/t/cant-deploy-to-pypi-anymore-pkg-resources-contextualversionconflict-importlib-metadata-0-18/10494/10
- https://lyftoss.slack.com/archives/CTZUQL0KX/p1605184474084500
Thanks to @lgomezma for finding the Travis link above.

**Comment by achantavy** [comment]
Temp fixed this in #461; waiting on https://github.com/travis-ci/dpl/issues/1222.

**Comment by achantavy** [comment]
Reverted with #471, see the notes there for details; we should be good now.
